.gitignore

```
@@ -124,3 +124,4 @@ safe_transaction_service/media
 
 # User-specific stuff:
 .idea/
+.vscode/
```

I think we can add `launch.json` and `settings.json` as part of the files to be tracked (else any update to those files will be ignored from now on)

```suggestion
.vscode/
!.vscode/launch.json
!.vscode/settings.json
```

---

That's the idea. When you configure vscode for yourself you will get into it some variables only meaninful to you (like the path for the virtualenvs or some other configurations). I don't want to be ignoring `.vscode` folder in every commit I do. This way if someone wants to update `.vscode` folders it can be done, but it needs to be added using `git add -f`

---

Updated the description

---

But this doesn't affect that other paths under `.vscode` (`.vscode` continues to be excluded). It just says that any updates to `launch.json` and `settings.json` won't be ignored or do we want them to be ignored?

---

We want them to be ignored. All the user changes will be on `settings.json` and `launch.json`, not on different files